Source/WebCore/platform/mediastream/mac/CoreAudioCaptureSource.cpp

```diff
@@ -846,8 +846,8 @@ const RealtimeMediaSourceSettings& CoreAudioCaptureSource::settings()
         RealtimeMediaSourceSettings settings;
         settings.setVolume(volume());
         settings.setSampleRate(sampleRate());
-        settings.setDeviceId(hashedId());
-        settings.setLabel(name());
+        settings.setDeviceId(AtomString { hashedId() });
+        settings.setLabel(AtomString { name() });
```

Comment:
Should we change `RealtimeMediaSource::name()` to be an `AtomString` instead?

Reply:
Yes, I am checking but I think so.